storage/imx6ull-flashnor/Makefile

```diff
@@ -4,10 +4,18 @@
 # Copyright 2021 Phoenix Systems
 #
 
-NAME := imx6ull-flashnor
-LOCAL_SRCS := flashnor.c flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor/nor.c
-LOCAL_HEADERS := flashnor-ecspi.h flashnor-qspi.h flashnor-drv.h nor/nor.h
+
+NAME := libflashnordrv-imx6ull
+LOCAL_SRCS := flashnor-ecspi.c flashnor-qspi.c flashnor-drv.c nor.c
+LOCAL_HEADERS := imx6ull-flashnor-ecspi.h imx6ull-flashnor-qspi.h imx6ull-flashnor-drv.h imx6ull-nor.h
```

Comment:
`DEP_LIBS` is not a valid parameter for `static-lib.mk`, maybe you meant `DEPS`? 

https://github.com/phoenix-rtos/phoenix-rtos-build/blob/master/makes/static-lib.mk#L10